README.md

```diff
@@ -6,81 +6,74 @@ Note that over time, this job, and all the jobs called by it, will be migrated t
+    [quay.io/eclipse/che-postgres](https://quay.io/repository/eclipse/che-postgres?tab=tags),
+    [quay.io/eclipse/che-dev](https://quay.io/repository/eclipse/che-dev?tab=tags),
+    [quay.io/eclipse/che-server](https://quay.io/repository/eclipse/che-server?tab=tags),
+    [quay.io/eclipse/che-dashboard-dev](https://quay.io/repository/eclipse/che-dashboard-dev?tab=tags) and
+    [quay.io/eclipse/che-e2e](https://quay.io/repository/eclipse/che-e2e?tab=tags)
+
+In the case of Che Operator, as well as workflows that depend on it - chectl, che-docs and community-operator PR generation. This is due to performing manual verifications in Che Operator by the Deploy team (and also various tests run against running Che, so we have a chance to see if it functions). When everything has been verified, after the merging of operator PRs the following projects workflows will be triggered automatically.
+- [chectl](https://github.com/che-incubator/chectl - release arifact is binary, which is also published on [Releases page]https://github.com/che-incubator/chectl/releases 
+- [che-docs](https://github.com/eclipse/che-docs) - only create tag and pull request to update to latest released version of Che
+- [community-operator](https://github.com/operator-framework/community-operators/) - create pull requests to https://github.com/operator-framework/community-operators/pulls?q=%22Update+eclipse-che+operator%22+is%3Aopen to update to latest released version of Che
+
+## Release phases
+
+At the moment, [Release - Orchestrate Overall Release Phases]((https://github.com/eclipse/che-release/actions?query=workflow%3A%22Release+-+Orchestrate+Overall+Release+Phases%22)) job has the way of ordering the release by utilizing the concept of phases.
+Currently there are several phases, representing an order of projects, which we can execute in parallel, as long as their dependent projects have been released. Projects in lower phases are those, on which projects from higher phase will depend.
+Phase 1 - [che-devfile-registry](https://github.com/eclipse/che-devfile-registry), [che-theia](https://github.com/eclipse/che-theia), [che-machine-exec](https://github.com/eclipse-che/che-machine-exec), [che-jwt-proxy](https://github.com/eclipse/che-jwtproxy), [kubernetes-image-puller](https://github.com/che-incubator/kubernetes-image-puller), [devworkspace-operator](https://github.com/devfile/devworkspace-operator), [che-dashboard](https://github.com/eclipse/che-dashboard)
+Phase 2 - [che-plugin-registry](https://github.com/eclipse/che-plugin-registry) - depends on [che-theia](https://github.com/eclipse/che-theia)
+Phase 3 - [che](https://github.com/eclipse/che) - depends on - [che-dashboard](https://github.com/eclipse/che-dashboard)
+Phase 4 - [devworkspace-che-operator](https://github.com/che-incubator/devworkspace-che-operator) - depends on [devworkspace-operator](https://github.com/devfile/devworkspace-operator)
```

Comment:
doesn't this also depend on devworkspace che operator, at least for now? (not sure we need to mention it if it's going away though...)

Reply:
haha, reading skills on my part. never mind